api/proto/core/dicehub/extension/extension.proto

```diff
@@ -8,24 +8,45 @@ import "google/protobuf/struct.proto";
 
 service ExtensionService {
     rpc SearchExtensions (ExtensionSearchRequest) returns (ExtensionSearchResponse)  {
+        option (google.api.http) = {
```

Comment:
This is just a statement that provides HTTP services, not OpenAPI

Reply:
I noticed that there is a statement of these interface under  `modules/openapi/api/apis/dicehub`. So maybe this HTTP services should be added to opanapi by protobuf option `option (erda.core.common.openapi)` instead of apiSpec.

Reply:
The old way to expose the openapi code has not been deleted, so no option (erda.core.common.openapi) is added to the proto.

![image](https://user-images.githubusercontent.com/28723047/161928503-6aaf4209-6574-4cc7-b008-9589a7bbb514.png)

It is indeed possible to delete these old exposed openapi codes and replace them with what you said

Reply:
![image](https://user-images.githubusercontent.com/28723047/161931883-396fecb6-fad8-414e-b4a7-701feb53973a.png)

![image](https://user-images.githubusercontent.com/28723047/161931923-87a4ec92-888f-4a78-8fca-1480c770df6a.png)

The same api permission verification is still different, and the new method may not be possible

![image](https://user-images.githubusercontent.com/28723047/161932266-8907bfb9-7ddf-4f9c-9765-1bec352ce944.png)

Reply:
of course you can.
see `api/proto/common/openapi.proto:8`.

Reply:
done